src/MediaWiki/Hooks/SidebarBeforeOutput.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace SMW\MediaWiki\Hooks;
+
+use SMW\MediaWiki\HookListener;
+use SMW\NamespaceExaminer;
+use SMWInfolink as Infolink;
+use Skin;
+use SpecialPage;
+use Title;
+use SMW\OptionsAwareTrait;
+/**
+ * Called at the end of Skin::buildSidebar().
+ *
+ * @see https://www.mediawiki.org/wiki/Manual:Hooks/SidebarBeforeOutput
+ *
+ * @license GNU GPL v2+
+ * @since 3.2
```

Comment:
Please drop the since tags, they are incorrect

Reply:
Dropped in the latest commit